packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -283,3 +284,13 @@ export function parseEnvironment(values: RAW_ENVIRONMENT): ENVIRONMENT {
 
   return environment;
 }
+
+/**
+ * Parse environment in node or browser without
+ * populating default values.
+ */
+export function parseEnvWithoutDefaults(): ENVIRONMENT {
```

Comment:
Nit: It would be nice to name this as `getEnvWithoutDefaults` as it is an alternative to `getEnv` rather than `parseEnvironment` -- `parseEnvironment` requires a parameter while `getEnv` is not.

Reply:
Sure thing. It's done.